### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by Mihndim2020**
This project is a clone of the New times article website (https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0). The objective of the project is the implement(put into use, showcase, cement) the skills learnt in the technical curriculum of microverse, which has to do with the positioning of elements using the various positioning techniques. The positing techniques used for this project are: floats, and flex-box, relative, absolute, fixed. Additional description about the project and its features.

### CODE REVIEW

**Comment by Mihndim2020** [comment]
Please  I do need help on how to go about the linters

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi Team     
#### Great Job so far, everything works fine, but there are a few more things required. 
### General
- [ ] You have duplications in your readme.MD file , let's fix it.
- [ ] Lets fix linter errors
- [ ] Kindly remove this file `git` as it is not needed for this project
- [ ] Kindly separate CSS from HTML.
- [ ] Do not make any changes in config files - they represent style guidelines that you share with your team 
- [ ] Lets fix the horizontal scroll
- [ ] Lets use [`<aside>` tag](https://www.w3schools.com/tags/tag_aside.asp) for `Editor's picks` section on the righmost side. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/aside_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/))
- [ ] Lets work on this 


# Page Structure
##### Bottom part

* [ ] Uses an anchor tag, any valid value e.g. "#" is okay for it's `href` attribute, to other articles with image previews on the leftmost side ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))
* [ ] Uses an anchor tag, any valid value e.g. "#" is okay for it's `href` attribute, to more popular articles on the rightmost side ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/positioning_and_floating_requirements.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/))

* [ ] Check usage of [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp) elements.e.g Footer (as described in Odin project description) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/semantic_html_requirement.png) from Pathwright)

### Make changes and submit another code review request.
*Happy coding* 
(Any questions Feel free to contact me on slack @kobiyoyo)

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Request Changes

Please remove linter errors and then request for a code review. 

Best,
Arslan Bisharat 
Slack:arslanbisharat

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,434 @@
...
+}
+
+.inflation-flex-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  width: 100%;
+  height: 90%;
+
+  /* background-color: red; */
```
Comment (umairarshadbutt) [inline css/style.css]: Remove commented CSS

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,434 @@
...
+}
+
+.subcontainer {
+  display: flex;
+  justify-content: space-around;
+  width: 100%;
+  height: 47%;
+
+  /* background-color: green; */
```
Comment (umairarshadbutt) [inline css/style.css]: Remove commented CSS

File: css/reset.css [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+/* section {
```
Comment (umairarshadbutt) [inline css/reset.css]: Remove commented CSS

File: index.html [hunk 4]
```diff
@@ -0,0 +1,602 @@
...
+                <span class="rick"> Rick Friedman for The New York Times </span>
+              </span>
+            </figcaption>
+          </figure>
+        </div>
+    </div>
+      
+      <section class="cambridge-section">
+        <article class="cambridge">
+            <h2> <a href="" class="author">By Dennis Overbye</a></h2>
```
Comment (umairarshadbutt) [inline index.html]: Please add `#` in href attribute

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @Mihndim2020
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [ ] Remove Commented CSS from css file. ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright)
- [ ] Please add `#` in href attribute
##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

File: index.html (hunk 4 above)
Comment (Mihndim2020) [reply index.html]: changes updated as requested

Comment (Mihndim2020) [reply index.html]: Updated

File: css/style.css (hunk 1 above)
Comment (Mihndim2020) [reply css/style.css]: Updated

File: css/style.css (hunk 2 above)
Comment (Mihndim2020) [reply css/style.css]: Updated

File: css/reset.css (hunk 3 above)
Comment (Mihndim2020) [reply css/reset.css]: Updated

File: index.html [hunk 5]
```diff
@@ -0,0 +1,602 @@
...
+                <div class="free-articles">
+                    <hr class="article-hr">
+                    <p class="free">Unlock more free articles.</p>
+                    <a href="#" class="free-article-anchor">Create an account of log in</a>
+                    <hr class="article-hr">
+                </div>
+            
+
+            <p>Reaching back across 13.8 billion years to <a href="" class="p-anchor">the first sliver of cosmic time</a> with telescopes at the South Pole, a team of astronomers led by John M. Kovac of the Harvard-Smithsonian Center for Astrophysics
+                detected ripples in the fabric of space-time — so-called <a href="" class="p-anchor">gravitational waves</a> — the signature of a universe being wrenched violently apart when it was roughly a trillionth of a trillionth of a trillionth
```
Comment (umairarshadbutt) [inline index.html]: Please add `#` in href attribute at two points before the merge

**Review by umairarshadbutt — APPROVED** [review]
# **Status: PROJECT APPROVED** :champagne:
Excellent job on this project guys. :fireworks: Keep it up.
##
**All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.**
- Congratulations on the splendid job @Mihndim2020 ! :+1: 
- You can now merge to master and submit the project completion form :rocket: . I wish you all the very best in your next tasks :1st_place_medal:
##
**If you have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

File: index.html (hunk 5 above)
Comment (Mihndim2020) [reply index.html]: Changes updated

**Pull request merged**